docs/features/where.md

```diff
@@ -7,4 +7,10 @@ parent: Base Features
 grand_parent: Features
 ---
 
-# Where
```

Comment:
I think you need to keep the top level heading or it won't show anywhere on the page what this page's topic is.

Reply:
ah, yeah, you're right! Sorry, VS Code yelled at me for having a top-level heading, so I got rid of it. I'll fix that.